### DISCUSSION TITLE
Indexierung films

### DISCUSSION THREAD

**Discussion by ls0218** [irgroup-classrooms/dis17-2021#13] [Q&A] [answered]
Hallo zusammen,

ich arbeite gerade an Exercise 2 (films) des Tutorials und habe alle Schritte befolgt. Leider werden mir nur 5 anstatt 1100 Results angezeigt. Ich bekomme beim Indexieren folgende Fehlermeldung angezeigt:
![image](https://user-images.githubusercontent.com/63605101/140914046-49768fdb-1174-4b97-b8a9-f7f3a4ad5725.png)

Hat noch jemand dieses Problem und kann mir ggf. weiterhelfen?

Vielen Dank im Voraus!

**Comment by xxjuly** [comment, marked as answer]
Lese dir nochmal diesen exakten Part des Tutorials durch: https://solr.apache.org/guide/8_10/solr-tutorial.html#create-the-names-field 
Er findet nur 5 Einträge, weil er "name" als float gecasted hat, da die ersten Einträge alles Zahlen sind für die Spalte. Wenn man im Schema das Feld als "text_general" einträgt kann der sowohl mit Namen bestehend aus Nummern als auch Buchstaben umgehen.

**Reply by ls0218** [reply]
Vielen vielen Dank!

**Reply by ls0218** [reply]
Es hat bei mir nur funktioniert wenn ich die Felder manuell im Admin Web Interface eintrage.

**Reply by lottehaupt** [reply]
![image](https://user-images.githubusercontent.com/63407473/141688154-84b1edb0-3947-4043-a61e-6f060c070d8d.png)
![image](https://user-images.githubusercontent.com/63407473/141688160-4d043bc9-feeb-482f-86c4-addaf02b6afe.png)
**hi, bei mir funktioniert das auch nicht. Hab das im cmd probiert und manuell bei Solr Admin. Die Felder existieren aber bei der Query **steht trotzdem nur 5 Beiträge....****
![image](https://user-images.githubusercontent.com/63407473/141688233-72e49d65-ca6c-4183-9590-e49bdcd49914.png)

**Reply by xxjuly** [reply]
Wahrscheinlich hat der sich jetzt automatisch ein falsches Schema gekrallt. Ich befürchte die schnellste Möglichkeit diesen Fehler loszuwerden ist einfach nochmal den kompletten Datensatz zu löschen, das Projekt neu anzulegen, im Schema das "name" Feld als text_general festzulegen (diesmal aber nicht über die cmd sondern DIREKT über den Browser), das catch all Feld anzulegen und dann die Indexierung mit einer der angegeben Code Schnippsel aus dem Tutorial zu starten.  Leider kann man das Schema nicht umändern wenn die Daten bereits indexiert sind, deswegen muss man die vorher rauslöschen, um sie dann mit dem richtigen Schema wieder zu indexieren. Da ich aber nicht weiß an welcher Stelle das Schema Probleme bereitet ist der Weg den ich genannt habe (auch wenn er aufwendiger ist) potenziell am erfolgreichsten.

**Reply by lottehaupt** [reply]
Danke für deine schnelle Antwort @xxjuly, es hat jetzt funktioniert! :)
